js/app.js

```diff
@@ -1 +1,291 @@
+				
+			}
+		}
+		winner()
+	});
+	t3.addEventListener('click', function(event) {
+		if (t3.textContent === "") {
+			if (turn % 2 === 0) {
+				t3.textContent = 'X';
+				gBoard.sq3= 'X';
+				turn ++;
+				document.getElementById('t3').style.color='blue';
+				
+			}
+			else {
+				t3.textContent = 'O';
+				gBoard.sq3= 'O';
+				turn ++;
+				document.getElementById('t3').style.color= "yellow";
+				
+			}
+		}
+		winner()
+	});
+	t4.addEventListener('click', function(event) {
+		if (t4.textContent === "") {
+			if (turn % 2 === 0) {
+				t4.textContent = 'X';
+				gBoard.sq4= 'X';
+				turn ++;
+				document.getElementById('t4').style.color='blue';
+				
+			}
+			else {
+				t4.textContent = 'O';
+				gBoard.sq4= 'O';
+				turn ++;
+				document.getElementById('t4').style.color= "yellow";
+				
+			}
+		}
+		winner()
+	});
+	t5.addEventListener('click', function(event) {
+		if (t5.textContent === "") {
+			if (turn % 2 === 0) {
+				t5.textContent = 'X';
+				gBoard.sq5= 'X';
+				turn ++;
+				document.getElementById('t5').style.color='blue';
+				
+			}
+			else {
+				t5.textContent = 'O';
+				gBoard.sq5= 'O';
+				turn ++;
+				document.getElementById('t5').style.color= "yellow";
+				
+			}
+		}
+		winner()
+	});
+	t6.addEventListener('click', function(event) {
+		if (t6.textContent === "") {
+			if (turn % 2 === 0) {
+				t6.textContent = 'X';
+				gBoard.sq6= 'X';
+				turn ++;
+				document.getElementById('t6').style.color='blue';
+				
+			}
+			else {
+				t6.textContent = 'O';
+				gBoard.sq6= 'O';
+				turn ++;
+				document.getElementById('t6').style.color= "yellow";
+				
+			}
+		}
+		winner()
+	});
+	t7.addEventListener('click', function(event) {
+		if (t7.textContent === "") {
+			if (turn % 2 === 0) {
+				t7.textContent = 'X';
+				gBoard.sq7= 'X';
+				turn ++;
+				document.getElementById('t7').style.color='blue';
+				
+			}
+			else {
+				t7.textContent = 'O';
+				gBoard.sq7= 'O';
+				turn ++;
+				document.getElementById('t7').style.color= "yellow";
+				
+			}
+		}
+		winner()
+	});
+	t8.addEventListener('click', function(event) {
+		if (t8.textContent === "") {
+			if (turn % 2 === 0) {
+				t8.textContent = 'X';
+				gBoard.sq8= 'X';
+				turn ++;
+				document.getElementById('t8').style.color='blue';
+				
+			}
+			else {
+				t8.textContent = 'O';
+				gBoard.sq8= 'O';
+				turn ++;
+				document.getElementById('t8').style.color= "yellow";
+				
+			}
+		}
+		winner()
+	});
+	t9.addEventListener('click', function(event) {
+		if (t9.textContent === "") {
+			if (turn % 2 === 0) {
+				t9.textContent = 'X';
+				gBoard.sq9= 'X';
+				turn ++;
+				document.getElementById('t9').style.color='blue';
+				
+			}
+			else {
+				t9.textContent = 'O';
+				gBoard.sq9= 'O';
+				turn ++;
+				document.getElementById('t9').style.color= "yellow";
+				
+			}
+		}
+		winner()
+	});
+
+
+newGame.addEventListener('click', function(event) {
+	newGame.textContent = location.reload();
+})
+
+function winner(){
```

Comment:
If you passed the current player into the winner() function, you could just test the win conditions for the player who just moved instead of both of them.